client/state/themes/themes-ui/test/reducer.js

```diff
@@ -11,7 +11,13 @@ import { expect } from 'chai';
 import reducer from '../reducer';
 
 describe( 'reducer', () => {
+	const state = reducer( undefined, {} );
+
 	test( 'should default to a backPath of /themes', () => {
-		expect( reducer( undefined, {} ) ).to.eql( { backPath: '/themes' } );
+		expect( state.backPath ).to.eql( '/themes' );
+	} );
+
+	test( 'should default to a themesBannerVisible of true', () => {
+		expect( state.themesBannerVisible ).to.eql( true );
```

Comment:
Let's add tests for the `SERIALIZE` and `DESERIALIZE` actions (grep for those in other test files for inspiration). That might also help us discover why we're not successfully persisting state for this reducer.